src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
```

Comment:
I think this class is generally well written! It will be good if you can include a short javadoc at the top so its easier for you if you were to revisit this project next time!